views/repo.html

```diff
@@ -67,13 +77,20 @@ <h3 class="panel-title">Deployment Badge</h3>
                 <p>Embed this badge on the project's README or on related tutorials and blog posts.</p>
                 <div class="input-group input-group-sm">
                   <span class="input-group-addon" id="image-url">Image URL:</span>
-                  <input type="text" class="form-control" readonly="readonly" value="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg" aria-describedby="image-url">
+                  <input type="text" class="form-control" readonly="readonly" value="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg" aria-describedby="image-url" />
+                  <span class="input-group-addon copy">
+                    <span class="glyphicon glyphicon-copy" data-clipboard-text="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg"></span>
+                  </span>
+                </div>
```

Comment:
There's a stray end tag `div` here which should be removed. This is causing the page to render incorrectly.